src/main/java/com/travelwith/servertravelwith/security/SecurityConfiguration.java

```diff
@@ -0,0 +1,23 @@
+/*
+ * Copyright (c) 2018. 
+ * Travel With
+ * Marcin Witek
+ */
+
+package com.travelwith.servertravelwith.security;
+
+import org.springframework.context.annotation.Configuration;
+import org.springframework.security.config.annotation.web.builders.HttpSecurity;
+import org.springframework.security.config.annotation.web.configuration.WebSecurityConfigurerAdapter;
+
+@Configuration
+public class SecurityConfiguration extends WebSecurityConfigurerAdapter {
+
+
+    @Override
+    protected void configure(HttpSecurity http) throws Exception {
+        http.authorizeRequests()
+                .antMatchers("/users/**").permitAll();
+        http.csrf().disable();
```

Comment:
Czemu to jest wyłączone ? csrf().disable();

Reply:
na razie chcialem to wylaczyc zeby sobie nie zaprzatac glowy z security. Jak zaimple,emtuje logowanie to wlacze xd